contracts/serverless-v2/Executors.sol

```diff
@@ -442,7 +442,16 @@ contract Executors is
     function _releaseExecutor(
         address _executor
     ) internal {
-        _postJobUpdate(_executor);
+        if(!executors[_executor].draining) {
+            // add node to the tree if its max job capacity was reached, else update the node
```

Comment:
please update the comment -> "If stakes are greater than minStakes then update the stakes for executors in tree if it already exists else add with latest stake."

Reply:
This is what I wanted to convey - the node might have been deleted due to max job capacity reached. So, if this is the case and updatedStakes >= MIN_STAKE_LIMIT, then we will insert the node, else update the stake value.

Will add the a suggested comment